stix_shifter_modules/qradar/tests/stix_translation/test_qradar_json_to_stix.py

```diff
@@ -135,11 +135,17 @@ def test_cybox_observables(self):
         assert(curr_obj.keys() == {'type', 'name'})
         assert(curr_obj['name'] == file_name)
 
+        proc_obj = TestTransform.get_first_of_type(objects.values(), 'process')
+        assert(proc_obj is not None), 'process object type not found'
+        assert(proc_obj.keys() == {'type', 'creator_user_ref'})
+        user_ref = proc_obj['creator_user_ref']
+        assert(user_ref in objects), f"creator_user_ref with key {proc_obj['creator_user_ref']} not found"
```

Comment:
an unittets that represents process object that reference username would be ideal when we add new mapping of such instead of `not found`

Reply:
Thats what the updated unit test does - it checks that process object exists, and that it has a valid reference to the user. Am I missing something?

Reply:
I'm confused. you don't have any process object but you are still creating one with user reference. not sure what you are trying to map.

Reply:
```
{"sourceip": source_ip, "destinationip": destination_ip, "url": url, "eventpayload": payload, "username": user_id, "protocol": 'TCP', "sourceport": "3000", "destinationport": 2000, "filename": file_name, "domainname": domain, "sourcemac": source_mac, "destinationmac": destination_mac}
```
this is the json results that it is trying to translate with your new mapping,